bridge/src/observers/nine-chronicles.ts

```diff
@@ -469,4 +503,22 @@ export class NCGTransferredEventObserver
             });
         }
     }
+
+    getAddressType(
+        sender: string,
+        recipient: string
+    ): ACCOUNT_TYPE | undefined {
```

Comment:
It might occured logical errors since determine maximum NCG by checking just it's type is Whitelist or not.
Thanks for your catch😀

Fixed not to return undefined at `getAddressType` func